regolith/schemas.py

```diff
@@ -1169,12 +1169,12 @@
         "month": {
             "description": "the month when the entry was created",
             "required": False,
-            "type": "integer",
+            "anyof_type": ["integer", "string"],
         },
         "year": {
             "description": "the year when the entry was created",
             "required": False,
-            "type": "integer",
+            "anyof_type": ["integer"],
```

Comment:
Sorry about that-to clarify, I actually thought about changing it back to type:"integer", but then thought the anyof_type would be better as it could be more easily changed in the future. I'll make the appropriate change now.